diff --git b/src/main/java/com/twilio/twiml/Play.java a/src/main/java/com/twilio/twiml/Play.java
index f86f93d..76a1892 100644
--- b/src/main/java/com/twilio/twiml/Play.java
+++ a/src/main/java/com/twilio/twiml/Play.java
@@ -14,7 +14,7 @@ public class Play extends TwiML {
     private final Integer loop;
 
     @XmlAttribute
-    private final Integer digits;
+    private final String digits;
 
     @XmlValue
     private final String body;
@@ -34,7 +34,7 @@ public class Play extends TwiML {
         return loop;
     }
 
-    public Integer getDigits() {
+    public String getDigits() {
         return digits;
     }
 
@@ -44,7 +44,7 @@ public class Play extends TwiML {
 
     public static class Builder {
         private Integer loop;
-        private Integer digits;
+        private String digits;
         private String body;
 
         public Builder(String body) {
@@ -56,7 +56,7 @@ public class Play extends TwiML {
             return this;
         }
 
-        public Builder digits(int digits) {
+        public Builder digits(String digits) {
             this.digits = digits;
             return this;
         }
diff --git b/src/test/java/com/twilio/twiml/PlayTest.java a/src/test/java/com/twilio/twiml/PlayTest.java
index 9c13e52..7154985 100644
--- b/src/test/java/com/twilio/twiml/PlayTest.java
+++ a/src/test/java/com/twilio/twiml/PlayTest.java
@@ -11,20 +11,20 @@ public class PlayTest {
     @Test
     public void testXml() throws TwiMLException {
         Play play = new Play.Builder("play this!")
-            .digits(5)
+            .digits("5w")
             .loop(3)
             .build();
 
-        Assert.assertEquals("<Play loop=\"3\" digits=\"5\">play this!</Play>", play.toXml());
+        Assert.assertEquals("<Play loop=\"3\" digits=\"5w\">play this!</Play>", play.toXml());
     }
 
     @Test
     public void testUrl() throws TwiMLException {
         Play play = new Play.Builder("play this!")
-            .digits(5)
+            .digits("5w")
             .loop(3)
             .build();
 
-        Assert.assertEquals("%3CPlay+loop%3D%223%22+digits%3D%225%22%3Eplay+this%21%3C%2FPlay%3E", play.toUrl());
+        Assert.assertEquals("%3CPlay+loop%3D%223%22+digits%3D%225w%22%3Eplay+this%21%3C%2FPlay%3E", play.toUrl());
     }
 }
